electrical/gps.py

```diff
@@ -4,7 +4,7 @@
 import time
 import pynmea2
 import csv
-if False: 
+if True: # import UbloxGPS if this is not a simulation
```

Comment:
Change this to `if False` because otherwise test cases on our laptop will fail. In the future, we want to avoid hardcoding this (onboarding task).